CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 - [User] Forget connection tokens on password change or on demand [#4214](https://github.com/vatesfr/xen-orchestra/issues/4214) (PR [#4224](https://github.com/vatesfr/xen-orchestra/pull/4224))
 - [Settings/Logs] LICENCE_RESTRICTION errors: suggest XCP-ng as an Open Source alternative [#3876](https://github.com/vatesfr/xen-orchestra/issues/3876) (PR [#4238](https://github.com/vatesfr/xen-orchestra/pull/4238))
 - [VM/Migrate] Display VDI size on migrate modal [#2534](https://github.com/vatesfr/xen-orchestra/issues/2534) (PR [#4250](https://github.com/vatesfr/xen-orchestra/pull/4250))
+- [SR/Create] Add API to create File type SR and to discover ZFS pools on an host [#4260](https://github.com/vatesfr/xen-orchestra/issues/4260) (PR [#4258](https://github.com/vatesfr/xen-orchestra/pull/4258))
```

Comment:
Do not add entries for server-side changes (because not relevant to users), we'll do it in the UI PR :slightly_smiling_face: